winui/Rating/Overview.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: About WinUI Rating control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
```

Comment:
description: Learn all about the introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more here.

Reply:
Changed the overview content description as mentioned above.